### ISSUE TITLE
Typings for componentized browser package

### ISSUE THREAD

**Issue by skh-** [rollbar/rollbar.js#880]
This [release with components](https://github.com/rollbar/rollbar.js/releases/tag/v2.19.0) is great!

Can we get typings for the individual components?  For example, `import Rollbar from 'rollbar/src/browser/core';` does not have a type.

We can kind-of work around this by setting it to the base rollbar type in a d.ts file, eg.

```
declare module 'rollbar/src/browser/core' {
  import Rollbar from 'rollbar';

  export default Rollbar;
}
```

However, this is not accurate (eg. `setComponents` is missing) and unsafe (since some features aren't available in the component.

**Comment by waltjones** [comment]
@skh-  Yes. :+1:

**Comment by waltjones** [comment]
Released in v2.19.4.
